index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <title>Medical core</title>
+</head>
+<body class="container">
+        <nav class="navbar">
+        <h1 class="navbar__logo">Medical core</h1>
+        <ul class="menu">
+            <li class="menu__item">Home</li>
+            <li class="menu__item">Product</li>
+            <li class="menu__item">Pricing</li>
+            <li class="menu__item">Contact</li>
+        </ul>
+        <ul class="auth">
+            <li class="auth__login">Login</li>
+            <li class="auth__signup button">JOIN US <span class="material-symbols-outlined">
+                arrow_forward
+                </span></li>
+        </ul>
+        </nav>
+      <main>
+         <section class="section-1">
+            <p class="section-1__welcome">Welcome</p>
+            <p class="section-1__description">A Great place to Receive Care</p>
+            <p class="section-1__description-2">Get your best looking smile</p>
+            <div class="">
+                <div class="">
+                    <div class="section-1__buttons">
+    <a class="button">Join Us</a>
+    <a class="button--active">Learn more</a></div></div>
+            </div>
+         </section>
+         <section class="section-2">
+            <img src="img/doctor.png" alt="">
+            <div class="expert">
+              <span class="expert__bar"></span>
+              <h1 class="expert__title">Meet Our Experts</h1>
+              <p class="expert__description">The problem trying to resolve the conflict between the two major realms of Classical Physics:</p>
+              <p class="expert__description">Newtonian mechanics</p>
+              <a href="" class="expert__link">Learn More <span class="material-symbols-outlined">
```

Comment:
If you create a btn block you can reuse it here and create a modifier on button instead of using material-symbols-outlined class, it look like modifier.

Reply:
material-symbols-outlined is a class for google icons. If I put different classes the icon will not be displayed